include/drivers/circuits/hawickcircuits_driver.h

```diff
@@ -0,0 +1,69 @@
+/*PGR-GNU*****************************************************************
+File: circuits_driver.h
+
+Generated with Template by:
+Copyright (c) 2022 pgRouting developers
+Mail: project@pgrouting.org
+
+Function's developer:
+Copyright (c) 2022 Nitish Chauhan
+Mail: nitishchauhan0022@gmail.com
```

Comment:
Just for security reasons
nitishchauhan0022@gmail.com -> nitishchauhan0022 at gmail.com

Reply:
ok, I will make all the changes.